blocks/editable/index.js

```diff
@@ -72,6 +76,39 @@ function getFormatProperties( formatName, parents ) {
 
 const DEFAULT_FORMATS = [ 'bold', 'italic', 'strikethrough', 'link' ];
 
+/**
+ * Transforms internal block's representation into an Element.
+ *
+ * @param {Array} value Value to transform
+ * @return {WPElement} Element.
+ */
+function valueToElement( value ) {
+	if ( ! Array.isArray( value ) ) {
+		return value;
+	}
+
+	return value.map( ( element, i ) => {
+		if ( typeof element === 'string' ) {
+			return element;
+		}
+
+		const [ type, props, ...children ] = element;
+		const reactProps = Object.keys( props ).reduce( ( accumulator, key ) => {
+			const mapping = {
+				class: 'className',
```

Comment:
It looks like adding those 2 mappings is enough to satisfy React.

Reply:
Okey, style needs its own handling, because I see this: https://reactjs.org/docs/error-decoder.html?invariant=62&args[]=

Reply:
All keys must go camel case, too.